crmsh/bootstrap.py

```diff
@@ -1876,6 +1865,48 @@ def join_csync2(seed_host):
     status_done()
 
 
+def join_ssh_merge(_cluster_node):
```

Comment:
I guess that this change recovers some older code, but the name of the method doesn't look really accurate.
Let's skip this issue anyway XD